data-processed/ERDC-SEIR/metadata-ERDC-SEIR.txt

```diff
@@ -0,0 +1,41 @@
+team_name: US Army Engineer Research and Development Center
+team_abbr: ERDC
+institution_affil: DoD
+team_contact: glover.e.george@erdc.dren.mil
+team_funding: US Army Geospatial Task Force
+team_experience: 
+
+model_name: SEIR
```

Comment:
This is not the problem with the PR not starting, but we would like to ask you to create a unique model_name, so that there is not confusion with other SEIR models already in the repo.

Reply:
Updated in master